firstvoices/backend/search/utils/validators.py

```diff
@@ -148,3 +150,23 @@ def get_valid_site_feature(input_site_feature_str):
     if len(selected_values) == 0:
         return None
     return selected_values
+
+
+def get_valid_site_slugs(input_site_slug_str, user):
+    if not input_site_slug_str:
+        return ""
+
+    input_site_slugs = input_site_slug_str.split(",")
+    valid_site_slugs = filter_by_viewable(user, Site.objects.all()).values_list(
+        "slug", flat=True
+    )
+    selected_values = []
+
+    for value in input_site_slugs:
+        cleaned_value = value.strip().lower()
+        if cleaned_value in valid_site_slugs and cleaned_value not in selected_values:
+            selected_values.append(cleaned_value)
```

Comment:
A simpler way to do this might be:

input_site_slugs = [value.strip().lower() for value in input_site_slug_str.split(",")]
selected_values = Site.objects().visible(user).filter(slug__in=input_site_slugs)